src/lib/wasm/MultipassStore.ts

```diff
@@ -620,6 +620,30 @@ class MultipassStore {
         }
     }
 
+    async importAccount(passphrase: string, settings?: { to?: Uint8Array; multipassBox?: wasm.MultiPassBox }): Promise<Result<WarpError, wasm.Identity | undefined>> {
+        let multipass = settings?.multipassBox ? settings.multipassBox : get(this.multipassWritable)
```

Comment:
this is because atm during import a temp warp instance is created
i guess we could just remove using the stored instance though since that atm is never used? thoughts?